app/src/main/java/org/simple/clinic/patient/recent/RecentPatientsView.kt

```diff
@@ -8,20 +9,37 @@ import com.jakewharton.rxbinding2.view.RxView
 import com.xwray.groupie.GroupAdapter
 import io.reactivex.Observable
 import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.rxkotlin.ofType
+import io.reactivex.subjects.PublishSubject
 import kotlinx.android.synthetic.main.recent_patients.view.*
 import org.simple.clinic.R
 import org.simple.clinic.activity.TheActivity
+import org.simple.clinic.summary.PatientSummaryCaller
+import org.simple.clinic.summary.PatientSummaryScreenKey
+import org.simple.clinic.util.UtcClock
 import org.simple.clinic.widgets.ScreenCreated
+import org.simple.clinic.widgets.UiEvent
 import org.simple.clinic.widgets.visibleOrGone
+import org.threeten.bp.Instant
+import java.util.UUID
 import javax.inject.Inject
 
 class RecentPatientsView(context: Context, attrs: AttributeSet) : FrameLayout(context, attrs) {
 
   @Inject
   lateinit var controller: RecentPatientsViewController
 
+  @Inject
+  lateinit var activity: TheActivity
+
   private val groupAdapter = GroupAdapter<com.xwray.groupie.ViewHolder>()
 
+  private val adapterEvents = PublishSubject.create<UiEvent>()
+
+  @Inject
+  lateinit var utcClock: UtcClock
```

Comment:
Can you group all properties that are annotated with `@Inject`?

Reply:
If it is absolutely necessary.

Reply:
That's the convention we've followed _everywhere_